data.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Holds the data associated with a website. Websites are identified by origin.
+ */
+class AkitaOriginData {
+  origin = null;
+  isMonetized = false;
+  // Object<AkitaPaymentPointerData>
+  paymentPointerData = {};
+
+  constructor(origin) {
+    this.origin = origin;
+  }
+
+  static fromObject(akitaOriginDataObject) {
+    const newThis = new AkitaOriginData(akitaOriginDataObject.origin);
+    newThis.isMonetized = akitaOriginDataObject.isMonetized;
+
+    for (const paymentPointer in akitaOriginDataObject.paymentPointerData) {
+      newThis.paymentPointerData[paymentPointer] = AkitaPaymentPointerData.fromObject(
+        akitaOriginDataObject.paymentPointerData[paymentPointer]
+      );
+    }
+
+    return newThis;
+  }
+
+  /**
+   *
+   * @param {{
+   *  paymentPointer: string,
+   *  assetCode?: string,
+   *  assetScale?: number,
+   *  amount?: number
```

Comment:
can you add a short description of what each of these is for?

Reply:
I've added an example and a link to the Web Monetization API docs which explains a bit more in depth. Hopefully more clear now.